src/main/java/seedu/address/logic/commands/DeleteCustomerOrderCommand.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.order.CustomerOrder;
+import seedu.address.model.order.OrderList;
+import seedu.address.model.product.PastryCatalogue;
+import seedu.address.model.product.Product;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+
+public class DeleteCustomerOrderCommand extends Command {
+    public static final String COMMAND_WORD = "deleteCustomerOrder";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Delete the customer order by their phone number. "
```

Comment:
perhaps breaking this line into separate lines will be better since it is a bit long?